gatsby-config.js

```diff
@@ -27,6 +28,83 @@ module.exports = {
                 includeInDevelopment: false,
             },
         },
+        {
+            resolve: `gatsby-plugin-feed`,
+            options: {
+                query: `
+                {
+                    site {
+                    siteMetadata {
+                        title
+                        siteUrl
+                    }
+                    }
+                }
+                `,
+                feeds: [
+                    {
+                        serialize: ({ query: { site, allSitePage } }) => {
+                            return allSitePage.edges.map(edge => {
+                                const path = dlv(edge, 'node.path');
+                                const query_fields = dlv(
+                                    edge,
+                                    'node.context._xrefDocMapping.query_fields'
+                                );
+                                const description = dlv(
+                                    query_fields,
+                                    [
+                                        'meta_description',
+                                        0,
+                                        'children',
+                                        0,
+                                        'value',
+                                    ],
+                                    'No description available'
+                                );
+                                const title = dlv(
+                                    query_fields,
+                                    ['title', 0, 'value'],
+                                    'No title available'
```

Comment:
If we don't have a title or description we should leave these blank (''). If this shows up in my rss reader with 'No title available' it's kind of weird. It's much more common to just have it missing.